rmbox/logger_convert.c

```diff
@@ -284,8 +284,10 @@ static int logger_read(struct convert_config *config,
 
 		/* checking log address */
 		if ((dma_log.address < snd->base_address) ||
-			dma_log.address > snd->base_address + snd->data_length)
+			dma_log.address > snd->base_address + snd->data_length) {
```

Comment:
Can you comment the logic here so no one needs to make any assumptions.

Reply:
updated